### PR TITLE
lndclient: add graph topology change subscription

### PR DESCRIPTION
**Pull request by bhandras**

### CODE REVIEW

File: lightning_client.go [hunk 1]
```diff
@@ -1879,25 +1949,25 @@ func (s *lightningClient) ChannelBackups(ctx context.Context) ([]byte, error) {
 	return resp.MultiChanBackup.MultiChanBackup, nil
 }
 
+// getOutPoint is a helper go convert a hash and output index to
+// a wire.OutPoint object.
+func getOutPoint(txId []byte, idx uint32) (*wire.OutPoint, error) {
```
Comment (guggero) [inline lightning_client.go:1954]: :+1:

File: lightning_client.go [hunk 2]
```diff
@@ -2680,6 +2750,150 @@ func (s *lightningClient) DescribeGraph(ctx context.Context,
...
+			return nil, err
+		}
+
+		result.ChannelEdgeUpdates[i] = ChannelEdgeUpdate{
+			ChannelID: lnwire.NewShortChanIDFromInt(
+				channelUpdate.ChanId,
+			),
+			ChannelPoint: *channelPoint,
+			Capacity:     btcutil.Amount(channelUpdate.Capacity),
+			RoutingPolicy: *getRoutingPolicy(
```
Comment (guggero) [inline lightning_client.go:2867]: nit: add a comment that this is always set in `lnd`'s `rpcserver.go` and therefore should never run into a nil reference error.

**Review by guggero — APPROVED** [review]
LGTM, nice addition :100:

**Comment by bhandras** [comment]
Since toplology change notifications are changing (will use feature map instead of serialized features) I'm putting this PR on hold.
More context: https://github.com/lightningnetwork/lnd/pull/4822

File: lightning_client.go (hunk 2 above)
Comment (bhandras) [reply lightning_client.go:2867]: Sorry, I pushed the merge button in the GitHub app for the wrong PR, will create a follow up (also making support for the new `Features` map).

**Pull request merged**